packages/front-end/components/PowerCalculation/stats.ts

```diff
@@ -0,0 +1,350 @@
+import normal from "@stdlib/stats/base/dists/normal";
+
+import {
+  PowerCalculationParams,
+  PowerCalculationResults,
+  MDEResults,
+  SampleSizeAndRuntime,
+  Week,
+} from "./types";
+
+/**
+ * delta method for relative difference
+ *
+ * @param varA Scalar control distribution variance.
+ * @param meanA Scalar control mean.
+ * @param nA Control sample size.
+ * @param varB Scalar treatment distribution variance.
+ * @param meanB Scalar treatment distribution mean.
+ * @param nB Treatment sample size.
+ * @param relative boolean indicator for relative effects.
+ * @returns variance.
+ */
+export function frequentistVariance(
+  varA: number,
+  meanA: number,
+  nA: number,
+  varB: number,
+  meanB: number,
+  nB: number,
+  relative: boolean,
+): number {
+  if (relative) {
+    return (
+      varB / (Math.pow(meanA, 2) * nB) +
+      (varA * Math.pow(meanB, 2)) / (Math.pow(meanA, 4) * nA)
+    );
+  } else {
+    return varB / nB + varA / nA;
+  }
+}
+
+export function powerStandardError(
+  variance: number,
+  mean: number,
+  n: number,
+  nVariations: number,
+  effectSize: number,
+): number {
+  return Math.sqrt(
+    frequentistVariance(
+      variance,
+      mean,
+      n / nVariations,
+      variance,
+      mean * (1 + effectSize),
+      n / nVariations,
+      true,
+    ),
+  );
+}
+
+export function calculateRho(
+  alpha: number,
+  sequentialTuningParameter: number,
+): number {
+  return Math.sqrt(
+    (-2 * Math.log(alpha) + Math.log(-2 * Math.log(alpha) + 1)) /
+      sequentialTuningParameter,
+  );
+}
+
+export function sequentialPowerSequentialVariance(
+  variance: number,
+  n: number,
```

Comment:
`n` here should be the total `n` for 2 variations at a time that are being compared (c.f. `confidence_interval` for Sequential test in gbstats), not the total N of the entire experiment. So I think you can leave this as is, but when you call it you want to do something like `2 * n / nVariations`